src/NUnitEngine/nunit.engine.core/Internal/TestAssemblyLoadContext.cs

```diff
@@ -12,6 +12,8 @@ namespace NUnit.Engine.Internal
 {
     internal sealed class TestAssemblyLoadContext : AssemblyLoadContext
     {
+        private static readonly Logger _log = InternalTrace.GetLogger(typeof(TestAssemblyLoadContext));
```

Comment:
Virtually all the other code uses `log` rather than `_log` for this even though it's technically not in accordance with the naming standards. To avoid surprise to other developers, I would use `log`.

Reply:
I selected name, looking on other class members. Seems you're using '_' prefix for them.

Reply:
Yes, that's technically the standard... but it's log in virtually all other files.

Reply:
I frankly don't see the difference. It's the same class member variable, like any other members, so I'd expect consistency in naming there. 
But if my assumptions are wrong - that's fine. If it's the requirements, but not suggestion - I can change the name from '_log' to 'log'

Reply:
It's not a choice between consistency and inconsistency but between two consistencies: (1) The observed practice that member variables start with '_' and (2) the use of `log.Debug`, `log.Info`etc. throughout the other files of the application. Beyond that, I don't believe there are any requirements except doing what makes sense to you.

As I already said, my comments are only comments. Only follow them if they make sense. :-)

Reply:
Well, might be "requirement" sounds too hard, but, it's always better when some person/or group of people maintains architectural consistency. 
Anyway, I checked other places, and there are "log" everywhere, so I renamed my loggers as well, and pushed a commit. Sorry for inattention.